kernel/vmemitcodes.ml

```diff
@@ -465,34 +465,14 @@ type body_code =
   | BCalias of Names.Constant.t
   | BCconstant
 
-type to_patch_substituted =
-| PBCdefined of to_patch substituted
-| PBCalias of Names.Constant.t substituted
-| PBCconstant
-
-let from_val = function
-| BCdefined tp -> PBCdefined (from_val tp)
-| BCalias cu -> PBCalias (from_val cu)
-| BCconstant -> PBCconstant
-
-let force = function
-| PBCdefined tp -> BCdefined (force subst_to_patch tp)
-| PBCalias cu -> BCalias (force subst_constant cu)
-| PBCconstant -> BCconstant
+type to_patch_substituted = body_code
+let from_val v = v
+let force v = v
```

Comment:
Is there a reason for not removing these types and functions?

Reply:
If we ever want to fiddle with them it's easier. Also, it breaks less API.

Reply:
Breaking less API is hardly a good reason here, since you removed these types and functions from `mod_subst.mli` anyway. It can hardly get worse at this point.

Reply:
OK, I'll remove it.

Reply:
Done.